samples/xbee/demos/EndToEndAgricultureSample/main.py

```diff
@@ -838,8 +839,7 @@ def upload_configuration_drm(configuration, sender=None):
         with datapoint_lock:
             datapoint.upload(data_stream, conf_value, data_type=datapoint.DataType.DOUBLE)
     except Exception as e:
-        print(e)
-        traceback.print_exc()
```

Comment:
There are more `traceback.print_exc()` in the code, we should take care of them too.
And if we are going to print the errors to the stderr instead of stdout, all errors should be in stderr.